duckietown_msgs/Pixel uv
---
duckietown_msgs/GroundCoord xy
